spec/feature/confirm_email_prompt_spec.rb

```diff
@@ -0,0 +1,110 @@
+RSpec.feature "Confirm email prompt" do
+  scenario "Seeing the banner on new user signup" do
+    given_i_newly_created_an_account
+    and_i_have_signed_in
+    when_i_navigate_to_home
+    then_i_see_the_confirmation_reminder_banner
+    when_i_navigate_to_manage
+    then_i_see_the_confirmation_reminder_banner
+    when_i_navigate_to_security
+    then_i_see_the_confirmation_reminder_banner
+  end
+
+  scenario "Resend from is prefilled on new user signup" do
+    given_i_newly_created_an_account
+    and_i_have_signed_in
+    when_i_navigate_to_home
+    then_i_see_the_confirmation_reminder_banner
+    when_i_click_the_link_on_the_confirmation_banner
+    then_i_see_the_new_confirmaton_page_header
+    and_my_email_address_should_be_prefilled_in_the_form
+  end
+
+  scenario "Resend from is prefilled for changed email address" do
+    given_i_requested_to_change_my_email_address
+    and_i_have_signed_in
+    when_i_navigate_to_home
+    then_i_see_the_confirmation_reminder_banner
+    when_i_click_the_link_on_the_confirmation_banner
+    then_i_see_the_new_confirmaton_page_header
+    and_my_confirmed_email_address_should_be_prefilled
+  end
+
+  scenario "Banner disappears after confirmation of email for new accounts" do
```

Comment:
As with the previous comment, this could be made slightly clearer, e.g. `Banner is not present once user has confirmed email address`

Reply:
agreed, much improved word-itude